AirBench/scripts/index.js

```diff
@@ -48,20 +50,89 @@
+            longitude = event.coordinate[1];
+            window.location.href = "/Bench/Create?Lat=" + latitude + "&Lon=" + longitude;
+        }
     });
 
-    async function print(){
-        console.log(latitude);
-        console.log(longitude);
+    // let createFilteredTable = function () {
+    //     const myTableDiv = document.getElementById("benches");
+    //     //Table
+    //     const table = document.createElement('TABLE');
+    //     table.id = 'table';
+    //     //THead
+    //     const tableHead = document.createElement('THEAD');
+    //     tableHead.className = 'thead-dark';
+    //     table.appendChild(tableHead);
+    //     //THead > TR
+    //     const tableHeadRow = document.createElement('TR');
+    //     tableHead.appendChild(tableHeadRow);
+    //     //THead > TR > TH
+    //     const tableHeadRowElement1 = document.createElement('TH');
+    //     tableHeadRowElement1.innerHTML = 'Bench No';
+    //     tableHeadRow.appendChild(tableHeadRowElement1);
+    //     const tableHeadRowElement2 = document.createElement('TH');
+    //     tableHeadRowElement2.innerHTML = 'Rating';
+    //     tableHeadRow.appendChild(tableHeadRowElement2);
+    //     const tableHeadRowElement3 = document.createElement('TH');
+    //     tableHeadRowElement3.innerHTML = 'Description';
+    //     tableHeadRow.appendChild(tableHeadRowElement3);
+    //     const tableHeadRowElement4 = document.createElement('TH');
+    //     tableHeadRowElement4.innerHTML = 'Seats';
+    //     tableHeadRow4.appendChild(tableHeadRowElement4);
+    //     //TBody
+    //     var tableBody = document.createElement('TBODY');
+    //     table.appendChild(tableBody);
+    //     for (var i = 0; i < 3; i++) {
+    //         var tr = document.createElement('TR');
+    //         tableBody.appendChild(tr);
+    //         for (var j = 0; j < 4; j++) {
+    //             var td = document.createElement('TD');
+    //             td.appendChild(document.createTextNode("Cell " + i + "," + j));
+    //             tr.appendChild(td);
+    //         }
+    //     }
+    //     myTableDiv.appendChild(table);
+    // };
+
+    function filterBenches(){
+        let min = parseInt(document.getElementById('min').value, 10);
+        let max = parseInt(document.getElementById('max').value, 10);
+        const tableInfo = document.getElementById('benches');
+        let rows = tableInfo.getElementsByTagName("tr");
+        console.log(rows)
+        for(let i = 1; i < rows.length; i++){
+            let seats = parseInt(rows[i].getAttribute('data-seats'), 10);
+
+            if(isNaN(min && isNaN(max))){
+                rows[i].style.display = "";
+            } 
+            else if (seats >= min && seats <= max) {
+                rows[i].style.display = "";
+            } else if (seats < min || seats > max) {
+                rows[i].style.display = "none";
+            } else {
+                rows[i].style.display = "";
+            }
+        }
```

Comment:
This looks a lot like Jason's filtering implementation. Collaboration is fine, but I also want to make sure that you understand how this code is working. Do you feel comfortable with how this code functions?